src/helm/benchmark/scenarios/mmlu_pro.py

```diff
@@ -0,0 +1,93 @@
+import csv
+import os
+from typing import Dict, List
+
+from helm.common.general import ensure_file_downloaded, ensure_directory_exists
+from helm.common.hierarchical_logger import hlog
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
```

Comment:
nit: use absolute imports rather than relative imports. In general, try to follow [PEP 8 style](https://peps.python.org/pep-0008/#imports) (even though our automatic linter doesn't enforce it).